flow-typed/types/text-style.js

```diff
@@ -1,11 +1,23 @@
 // @flow
+
+/**
+ * @typedef {Object} PKTextStyleObject
+ * @property {"50%" | "75%" | "100%" | "200%" | "3000%" | "4000%"} fontSize='100%' - Percentage unit relative to the parent element's font size.
+ * @property {-2 | -1 | 0 | 2 | 3 | 4} fontScale=0 - - Integer number representing the scaling factor relative to the parent element's font size..
```

Comment:
@JonathanTGold  please add note what are the defaults

Reply:
@giladna comments here, not in the flow type https://github.com/kaltura/playkit-js/blob/master/src/track/text-style.js